### PR TITLE
Update Docs - utilities.rst based on issue #195

### PR DESCRIPTION
**Pull request by georgemillard**
Having encountered this issue a couple of times with FieldTrackers, I thought it would be good to go in the docs! Hope this helps.

## Problem

Generic FieldTrackers track ForeignKey Fields by db_column name, but there is no mention of this in the docs.

Issue #195 

## Solution

Updated the docs.

## Commandments

- [ ] Write PEP8 compliant code.
- [ ] Cover it with tests.
- [ ] Update `CHANGES.rst` file to describe the changes, and quote according issue with `GH-<issue_number>`.
- [ ] Pay attention to backward compatibility, or if it breaks it, explain why.
- [x] Update documentation (if relevant).

### CODE REVIEW

File: docs/utilities.rst [hunk 1]
```diff
@@ -212,6 +212,50 @@ An example using the model specified above:
...
+
+    class Child(models.Model):
+        name = models.CharField(max_length=64)
+        parent = models.ForeignKey(Parent)
+
+.. code-block:: pycon
+    
+    >>> p = Parent.objects.create(name='P')
+    >>> c = Child.objects.create(name='C', parent=p)
+    >>> c.tracker.has_changed('parent_id')
```
Comment (kezabelle) [inline docs/utilities.rst]: There's no `tracker` attribute on the definition of `Child`

File: docs/utilities.rst [hunk 2]
```diff
@@ -212,6 +212,50 @@ An example using the model specified above:
...
+    from model_utils import FieldTracker
+
+    class Parent(models.Model):
+        name = models.CharField(max_length=64)
+
+    class Child(models.Model):
+        name = models.CharField(max_length=64)
+        parent = models.ForeignKey(Parent)
+
+.. code-block:: pycon
```
Comment (kezabelle) [inline docs/utilities.rst]: I presume you meant `python` rather than `pycon`

File: docs/utilities.rst [hunk 3]
```diff
@@ -212,6 +212,50 @@ An example using the model specified above:
...
+    
+    >>> p = Parent.objects.create(name='P')
+    >>> c = Child.objects.create(name='C', parent=p)
+    >>> c.tracker.has_changed('parent_id')
+    
+    
+To find the db_column names of your model (using the above example):
+
+.. code-block:: pycon
```
Comment (kezabelle) [inline docs/utilities.rst]: Same as above, `python` not `pycon` ?

File: docs/utilities.rst (hunk 2 above)
Comment (georgemillard) [inline docs/utilities.rst]: Hi, I followed the markdown conventions used in the previous section of the doc. They used pycon for console commands and python for code. I'm happy to amend both to python if that suits. Thanks

Comment (kezabelle) [inline docs/utilities.rst]: Nah you're quite right, I'd entirely forgotten it was a separate lexer. In my head it was just a typo.

Comment (georgemillard) [inline docs/utilities.rst]: cool, thanks

**Comment by jn8029** [comment]
@RDIL  this seems good to merge.

**Comment by RDIL** [comment]
I’m no longer in the organization. Sorry.

**Comment by Natim** [comment]
@georgemillard could you update the Child example definition to add a FieldTracker?

File: docs/utilities.rst (hunk 1 above)
Comment (georgemillard) [reply docs/utilities.rst]: Thanks, done :-)

**Pull request merged**